packages/peregrine/lib/talons/CategoryList/useCategoryList.js

```diff
@@ -32,8 +32,8 @@ export const useCategoryList = props => {
     }, [runQuery, setLoading, id]);
 
     return {
-        data,
         error,
-        loading
+        childCategories:
```

Comment:
This is a good change, in line with your other comment. Stylistically, I do think you should do calculations outside of the return block, but it's not a big deal.

Also I think we should still return `loading`, since a well-designed component would use it. Just because ours doesn't use it doesn't mean we should exclude it.